### PR TITLE
Adding custom duration that serializes nicely

### PR DESCRIPTION
**Pull request by larsbeck**
# Description

This adds a custom type `types.Duration` which the flag parsing library knows (we register it). Therefore when we use this type in options that are passed to the runner, this type's `MarshalJSON()` will be called and render a duration in a nice format rather than nanoseconds.

### CODE REVIEW

File: types/duration.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+import "time"
+
+// Duration is a wrapper around time.Duration that implements json.Marshaler.
+type Duration time.Duration
+
+// MarshalJSON implements json.Marshaler. It calls time.Duration.String() to
+// serialize the duration.
+func (d Duration) MarshalJSON() ([]byte, error) {
+	return []byte(`"` + time.Duration(d).String() + `"`), nil
```
Comment (sebastian-quintero) [inline types/duration.go]: Why the extra `"`?

**Review by sebastian-quintero — APPROVED** [review]

**Comment by sebastian-quintero** [comment]
We might need to change the `shift-scheduling` template with this PR to reflect the use of the new type?

File: types/duration.go (hunk 1 above)
Comment (larsbeck) [reply types/duration.go]: Good catch. Removed

**Comment by sebastian-quintero** [comment]
nextroute moved over to another repo.

**Pull request closed without merging**